data/xml/2025.tlt.xml

```diff
@@ -0,0 +1,195 @@
+      <url hash="293f397e">2025.tlt-1.15</url>
+      <bibkey>tercon-dobrovoljc-2025-comparatree</bibkey>
+    </paper>
+    <paper id="16">
+      <title><fixed-case>U</fixed-case>niversal <fixed-case>D</fixed-case>ependency Treebank for a low-resource Dardic Language: <fixed-case>T</fixed-case>orwali</title>
+      <author><first>Naeem</first><last>Uddin</last><affiliation>Charles University Prague</affiliation></author>
+      <author><first>Daniel</first><last>Zeman</last><affiliation>Faculty of Mathematics and Physics, Charles University Prague</affiliation></author>
+      <pages>140-147</pages>
+      <abstract>This paper presents and discusses the linguistic phenomena encountered in the development of the ongoing first ever universal dependency treebank for Torwali the Language. Torwali belongs to the Kohistani sub-group of Dardic Indo-Aryan languages, and is considered an endangered (Moseley, 2010) and indigenous language, which makes it extremely low-resourced in terms of linguistic and computational resources. With the aim of including Torwali in Universal Dependencies (UD) (de Marneffe et al. 2021), we are annotating a diverse set of example sentences for POS tags, features and dependency relations.</abstract>
+      <url hash="81a24126">2025.tlt-1.16</url>
+      <bibkey>uddin-zeman-2025-universal</bibkey>
+    </paper>
+    <paper id="17">
+      <title>Legal-<fixed-case>CGEL</fixed-case>: Analyzing Legal Text in the <fixed-case>CGELB</fixed-case>ank Framework</title>
+      <author><first>Brandon</first><last>Waldon</last><affiliation>University of South Carolina</affiliation></author>
+      <author><first>Micaela</first><last>Wells</last><affiliation>Georgetown University</affiliation></author>
+      <author><first>Devika</first><last>Tiwari</last><affiliation>Georgetown University</affiliation></author>
+      <author><first>Meru</first><last>Gopalan</last><affiliation>Georgia Institute of Technology</affiliation></author>
+      <author><first>Nathan</first><last>Schneider</last><affiliation>Georgetown University</affiliation></author>
+      <pages>148-153</pages>
+      <abstract>We introduce Legal-CGEL, an ongoing treebanking project focused on syntactic analysis of legal English text in the CGELBank framework (Reynolds et al., 2022), with an initial focus on US statutory law. When it comes to treebanking for legal English, we argue that there are unique advantages to employing CGELBank, a formalism that extends a comprehensive—and authoritative—formal description of English syntax (the _Cambridge Grammar of the English Language_; Huddleston &amp; Pullum, 2002). We discuss some analytical challenges that have arisen in extending CGELBank to the legal domain. We conclude with a summary of immediate and longer-term project goals.</abstract>
```

Comment:
Underscore markup (Markdown as supported by OpenReview) not converted to italics